File: cash.c
#import <stdio.h>
#import <cs50.h>
#import <math.h>

int main(void)
{
    float change_input;
    int change;
    int counter = 0;
    do
    {
        change_input = get_float("Change owed: ");
    }
    while (change_input < 0.009);
    change = round(change_input * 100);
    while (change > 0)
    {
        if (change - 25 >= 0)
        {
            change = change - 25;
            counter++;
        }
        else if (change - 10 >= 0)
        {
            change = change - 10;
            counter ++;
        }
        else if (change - 5 >= 0)
        {
            change = change - 5;
            counter ++;
        }
        else if (change - 1 >= 0)
        {
            change = change - 1;
            counter ++;
        }
    }
    printf("%i\n", counter);
}

File: pset6/cash/cash.py
while True:
    try:
        change = float(input("Change owed:"))
        if type(change) == float and change > 0.009:
            change = round(change*100)
            break
    except:
        continue
counter = 0
coins = [25, 10, 5, 1]
while change > 0:
    for i in coins:
        while change - i >= 0:
            change -= i
            counter += 1
print(counter)

